content/en/news/releases/1.10.x/announcing-1.10.1/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Announcing Istio 1.10.1
+linktitle: 1.10.1
+subtitle: Patch Release
+description: Istio 1.10.1 patch release.
+publishdate: 2021-06-08
```

Comment:
Based on earlier comments, I think this needs to be updated to June 9th.

Reply:
Done